flow-typed/hex2rgba.js

```diff
@@ -0,0 +1,3 @@
+declare module 'hex2rgba' {
+  declare module.exports: function;
```

Comment:
Can we declare a real type for this function? `(hex : string, alpha? : number) => string` I think.

Reply:
Yup! On it.